connectors/es/sink.py

```diff
@@ -444,7 +454,11 @@ async def get_docs(self, generator):
                         doc[TIMESTAMP_FIELD] = iso_utc()
 
                 # if we need to call lazy_download we push it in lazy_downloads
-                if self.content_extraction_enabled and lazy_download is not None:
+                if (
+                    self.optimize_on_download_timestamp
```

Comment:
Is the `optimize_on_download_timestamp` check needed here? This `if` statement affects both existing and new docs. The `optimize_on_download_timestamp` should only relate to existing docs. Would checking if `self.content_extraction_enabled and lazy_download is not None:` be sufficient here?

Reply:
Can disabling the optimisation prevent us from downloading any content?